lerna.json

```diff
@@ -1,4 +1,4 @@
 {
-  "packages": ["packages/*/*"],
+  "packages": ["packages/*/*", "native-packages/*/*"],
```

Comment:
This should probably be `native-packages/*`?

Reply:
```suggestion
  "packages": ["packages/*/*", "native-packages/*"],
```